.github/workflows/ci_tests.yaml

```diff
@@ -89,8 +89,8 @@ jobs:
       # Install GMT and other required dependencies from conda-forge
       - name: Install dependencies
         run: |
-          conda install -c conda-forge/label/dev gmt=6.2.0rc1
-          conda install numpy=${{ matrix.numpy-version }} \
+          conda install conda-forge/label/dev::gmt=6.2.0rc1 \
```

Comment:
I can't seem to find any mention of the double colon `::` way of specifying channels for a single package at https://docs.conda.io/projects/conda/en/latest/commands/install.html, but I know it works and the "Install dependencies" step passes at https://github.com/GenericMappingTools/pygmt/runs/2627248691?check_suite_focus=true#step:5:1, so ... it should be ok?